client/app/styles/main.less

```diff
@@ -387,12 +387,43 @@ h2 {
     }
   }
 
-  .shape {
+  .stack .shape .highlighted {
+    display: none;
+  }
+
+  .stack .onlyHighlight .shape  {
+    .border { display: none; }
+    .shadow { display: none; }
+    .node { display: none; }
+    .highlighted { display: inline; }
+  }
+
+  .stack .shape .metric-fill {
+    display: none;
+  }
+
+  .stack .onlyMetrics .shape .metric-fill {
+    display: inline-block;
+  }
 
+  .shape {
     /* cloud paths have stroke-width set dynamically */
     &:not(.shape-cloud) .border {
       stroke-width: @node-border-stroke-width;
       fill: @background-color;
+      transition: stroke-opacity 0.5s cubic-bezier(0,0,0.21,1), fill 0.5s cubic-bezier(0,0,0.21,1);
```

Comment:
What made you choose `cubic-bezier(0,0,0.21,1)`? In other places we use `ease-in-out`. Would be good to have transitions perform similarly to keep the *feel* the same. Maybe extract the transition style as a variable.